libs/algo/odc/algo/_masking.py

```diff
@@ -725,3 +725,28 @@ def _nodata_fuser(xx, **kw):
     if xx.shape[0] <= 1:
         return xx
     return choose_first_valid(xx, **kw)
+
+
+def _first_np(*aa: np.ndarray) -> np.ndarray:
+    return aa[0]
+
+
+def choose_first(x: xr.DataArray) -> xr.DataArray:
+    """
+    ``Out[0, y, x] = In[0, y, x]``
+
+    Expects data in ``_, y, x`` order. Works on Dask inputs too.
+    """
+
+    return _fuse_with_custom_op(
+        x, _first_np, name="choose_first"
+    )
+
+
+def _first_fuser(xx):
+
+    if isinstance(xx, xr.Dataset):
+        return xx.map(choose_first)
+    if xx.shape[0] <= 1:
+        return xx
+    return choose_first(xx)
```

Comment:
there is already choose_first_valid, just picking "first" regardless of `nodata` doesn't make sense.